relay_test.go

```diff
@@ -516,6 +516,26 @@ func TestRelayMakeOutgoingCall(t *testing.T) {
 	})
 }
 
+func TestConnContext(t *testing.T) {
```

Comment:
can we also add a test for the outbound connect case, where we create a connection from the relay to some other channel, and when that other channel sends a call, it'll use that relay-created connection and should have the context value set on the outbound side